client/layout/index.jsx

```diff
@@ -75,6 +78,23 @@ Layout = React.createClass( {
 		return sortBy( this.props.sites.get(), property( 'ID' ) ).pop();
 	},
 
+	renderMasterbar() {
+		if ( 'checkout' === this.props.section && abtest( 'checkoutMasterbar' ) === 'minimal' ) {
+			return (
+				<CartData>
+					<MasterbarMinimal selectedSite={ this.props.sites.getSelectedSite() } />
```

Comment:
Since this is highly tied to the checkout and the shopping cart, maybe we should name this component `<MasterbarCheckout />` or `<MasterbarLoggedInCheckout />` to make that more explicit. What do you think?